jx_sqlite/setop_table.py

```diff
@@ -169,14 +170,21 @@ def place(parent_doc_details):
 
                         if isinstance(s.value, LeavesOp):
                             for column in db_columns:
+                                if isinstance(column.nested_path, list):
+                                    column.nested_path=column.nested_path[0]
+                                if column.nested_path and column.nested_path!=nested_path:
+                                    continue                                   
```

Comment:
this logic looks like a mess is created somewhere else, and this is compensating for it.  Make sure `column.nested_path` is always a list.

`ColumnMapping = DataClass(` has a constraint 

    constraint={"and": [
        {"in": {"type": ["null", "boolean", "number", "string", "object"]}},
        {"gte": [{"length": "nested_path"}, 1]}
    ]}

maybe we should add it to `Column = DataClass(` too